controllers/idler/mapper.go

```diff
@@ -0,0 +1,19 @@
+package idler
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/types"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+// MapPodToIdler maps the pod to the idler
+func MapPodToIdler(_ context.Context, obj runtimeclient.Object) []reconcile.Request {
+	return []reconcile.Request{{
+		// the idler should have the same name as the user's namespace
+		NamespacedName: types.NamespacedName{
+			Name: obj.GetNamespace(),
```

Comment:
Does not specifying the namespace of the idler resource work correctly with the controller manager?